tests/integration/test_startup_scripts/configs/config.xml

```diff
@@ -0,0 +1,14 @@
+<clickhouse>
+    <startup_scripts>
+        <scripts>
```

Comment:
Shouldn't it be singular `<script>`, since each entry defines one script inside `<startup_scripts>` ?

Reply:
This is how it was initially done in a private build, so we should keep it as compatibility.